docker/test_generate_makefile.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generate Makefile test."""
+
+import io
+import sys
+
+from docker import generate_makefile
+
+
+def test_print_makefile_build():
+    """Tests result of a makefile generation for an image."""
+
+    name = 'afl-zlib-builder-intermediate'
```

Comment:
consider adding a test for a `-runner` image as well

Reply:
Done